.github/workflows/build-and-test-rh.yml

```diff
@@ -41,7 +41,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        image: [ 'centos:7', 'quay.io/centos/centos:stream9', 'fedora:35', 'fedora:36', 'fedora:38']
+        image: [ 'centos:7', 'quay.io/centos/centos:stream9', 'fedora:36', 'fedora:38', 'fedora:40']
```

Comment:
Should not we leave for Fedora only v39 and v40 as the previous ones already EOLed?

Reply:
Left Fedora 39, Fedora 40